fog/ledger/server/tests/router_connection.rs

```diff
@@ -612,7 +612,7 @@ fn fog_ledger_blocks_api_test(logger: Logger) {
         );
 
         // Try to get a block
-        let queries = [0..1];
+        let queries = [0..1; 1];
```

Comment:
This one comes from, https://rust-lang.github.io/rust-clippy/master/index.html#/single_range_in_vec_init